local-cookbooks/cyclescape/templates/default/passenger.vhost.conf

```diff
@@ -80,14 +80,22 @@
       RewriteCond %{HTTP_HOST} ^www\.(.+)\.cyclescape.org$ [NC]
       RewriteRule ^/(.*) https://%1.cyclescape.org/$1 [L,R]
 
+      SSLCertificateFile /etc/apache2/ssl/cert.pem
+      SSLCertificateKeyFile /etc/apache2/ssl/privkey.pem
+      SSLCertificateChainFile /etc/apache2/ssl/chain.pem
+
+      # From https://github.com/certbot/certbot/blob/d557475bb6f921b7bdb0459d77a162d673044dce/certbot-apache/certbot_apache/centos-options-ssl-apache.conf
+      # START
       SSLEngine on
-      SSLCertificateFile /etc/apache2/ssl/cyclescape-org.ssl.crt
-      SSLCertificateKeyFile /etc/apache2/ssl/cyclescape-org.ssl.nopassword.key
-      SSLCertificateChainFile /etc/apache2/ssl/ca.ssl.cer
 
-      SSLProtocol All -SSLv2 -SSLv3
-      SSLHonorCipherOrder On
-      SSLCipherSuite ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5:!DSS
+      # Intermediate configuration, tweak to your needs
+      SSLProtocol             all -SSLv2 -SSLv3
+      SSLCipherSuite          ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS
```

Comment:
I've blindly copied from the letsencrypt repo.  Does this seem ok @mvl22 ?

Reply:
Can you try the Mozilla Intermediate profile, at:
https://mozilla.github.io/server-side-tls/ssl-config-generator/

and try that on https://www.ssllabs.com/ssltest/ (please tick Do not show).

```
# intermediate configuration, tweak to your needs
SSLProtocol             all -SSLv3
SSLCipherSuite          ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS
SSLHonorCipherOrder     on
SSLCompression          off
```